doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
```

Comment:
It seems important to me to both (a) rationalize why a refactor is necessary (beauty counts!) and (b) track the effort involved so that we can more accurately predict/reflect our time estimates. Refactoring is like tests and documentation: there is really no end to them, just a stopping/pausing place, so it is both hard and important to track.

Whether the tracking mechanism is the story tracker is a different, tactical question. I don't have strong feelings about that.